flake.nix

```diff
@@ -106,6 +106,21 @@
                 pkgs.drivebrain_software
               ];
             };
+
+            devShells.tests = pkgs.mkShell rec {
+              name = "test-devshell";
+              shellHook = ''
+                  export PLUH="hello"
```

Comment:
prob dont need this shellhook for just this variable. prob will need shellhook for the generated dbc file tho